src/main/java/org/springframework/integration/aws/inbound/kinesis/KinesisMessageDrivenChannelAdapter.java

```diff
@@ -499,6 +529,51 @@ protected void doStart() {
 		}
 	}
 
+	private List<Shard> readShardList(String stream) {
+		return this.readShardList(stream, 0);
+	}
+
+	private List<Shard> readShardList(String stream, int retryCount) {
+		List<Shard> shardList = new ArrayList<>();
+
+		if (retryCount > this.describeStreamRetries) {
+			throw new IllegalStateException(
+					"Kinesis could not read shards from stream with name:[" + stream + "] ");
```

Comment:
Need space between `:` and `[`.
Although do we need a colon at all...

Reply:
Done